src/python.c

```diff
@@ -1257,16 +1257,34 @@ static int cpy_init_python(void) {
   Py_Initialize();
   python_sigint_handler = PyOS_setsig(SIGINT, cur_sig);
 
-  PyType_Ready(&ConfigType);
-  PyType_Ready(&PluginDataType);
+  if (PyType_Ready(&ConfigType) < 0) {
+    cpy_log_exception("python initialization");
```

Comment:
This and the below error messages are all the same, this makes debugging harder than necessary, please change these to include the actual failure in the error message so each error can be tracked to the respective line.